perses/app/setup_relative_calculation.py

```diff
@@ -921,6 +960,9 @@ def _resume_run(setup_options):
             total_steps = setup_options['n_cycles']
             run_so_far = simulation.iteration
             left_to_do = total_steps - run_so_far
+            # set context caches
+            simulation.sampler_context_cache = sampler_context_cache
```

Comment:
This should be `propagation_context_cache`, not `sampler_context_cache`.

Reply:
It is called `sampler_context_cache` on the openmmtools side, I used to call it `propagation_context_cache` but in some discussion this was changed. We could settle on one and make a bugfix release for `openmmtools` as well.